app/tray/Account/Account/Requests/TransactionRequest/TxMainNew/overview.js

```diff
@@ -105,13 +98,13 @@ const TxOverview = ({ req, chainName, chainColor, symbol, originName, txMeta, si
             style={{ background: valueColor }}
           >
             <div className='_txDescription'>
-              <TxDescription chain={chainName} chainColor={chainColor}>
+              <RequestChainHeader chain={chainName} chainColor={chainColor}>
```

Comment:
maybe something more generic that doesn't refer to "chain" here as there's other info in this header?